pyglmnet/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .pyglmnet import GLM, set_log_level
 from .utils import softmax, label_binarizer, log_likelihood
 from .datasets import fetch_tikhonov_data
-__version__ = '1.0.0'
+__version__ = '1.0.1'
```

Comment:
@jasmainak lmk if you think we should use another strategy for versioning. if not, i can merge and update our users!